binplace.targets

```diff
@@ -36,6 +36,7 @@
   <Target Name="GetTargetingPackDir">
     <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('NETCoreApp')) Or $(NuGetTargetMoniker.Contains('NETStandard')) Or $(NuGetTargetMoniker.Contains('UAP'))">
       <TargetingPackDir Include="$(RefPath)" />
+      <TargetingPackDir Condition="'$(TargetGroup)'!='$(_bc_TargetGroup)'" Include="$(RefRootPath)$(TargetGroup)/" />
```

Comment:
We should add `$(RefRootPath)$(TargetGroup)/` to that set in dir.props. 

For your question @tarekgh the others aren't needed in here yet. @ericstj will be tweaking the bin-placing logic at some point and they might become interesting at that point or perhaps some variants of them.